### ISSUE TITLE
CJK display problem

### ISSUE THREAD

**Issue by fundon** [dbcli/pgcli#243]
```
> SELECT * from goods_tags;
+------+--------+---------------------+---------------------+-----------+
|   id | name   | created_at          | updated_at          |   user_id |
|------+--------+---------------------+---------------------+-----------|
|    1 | bike   | 2015-05-26 10:53:43 | 2015-05-26 10:53:43 |         4 |
|    2 | 自行车    | 2015-05-26 11:56:32 | 2015-05-26 11:56:32 |         4 |
+------+--------+---------------------+---------------------+-----------+
SELECT 2
Command Time: 5.00679016113e-06
Format Time: 0.000900030136108

```

**Comment by amjith** [comment]
Thank you reporting the problem. It is now fixed in master. I'll be releasing the new version today. In the mean time if you'd like to try the new version you can try installing directly from master, as follows: 

```
pip install -U -e git+https://github.com/dbcli/pgcli.git#egg=pgcli
```

**Comment by fundon** [comment]
@amjith Thanks, LGTM

**Comment by amjith** [comment]
Available in the latest version 0.17.0. Please upgrade using: 

```
pip install -U pgcli
```
